libbeat/kibana/index.go

```diff
@@ -0,0 +1,121 @@
+package kibana
+
+import (
+	"encoding/json"
+	"errors"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"regexp"
+
+	"github.com/elastic/beats/libbeat/common"
+)
+
+type Index struct {
+	Version   string
```

Comment:
Reminds me a lot of `beat.Info` (https://github.com/elastic/beats/blob/master/libbeat/beat/info.go#L6) but Dir is missing there.